components/ContactList.js

```diff
@@ -26,19 +26,33 @@ import ContactItem from "../components/ContactItem";
 // Erin Larson – (542) 321-3456
 // .....
 
-export default function ContactList() {
+export default function ContactList({ addFavorites, removeFavorites }) {
   const dispatch = useDispatch();
   const contacts = useSelector(getContacts);
 
   useEffect(() => {
     dispatch(fetchContacts());
   }, []);
-
+   
+  const sorted = contacts.sort((a, b) => a.lastName.localeCompare(b.lastName))
+  .reduce((acc, val) => {
+    const key = val.lastName[0];
+    if(!acc[key]) acc[key] = []
+    acc[key].push(val);
+    return acc;
+  }, {})
```

Comment:
This is a nice sort function and I like the idea of converting contacts into an object with each letter section being a key.

However, you'd probably want to also check for a phone number in this sort function so that you don't end up with empty sections (for example: your app currently has an "S" section without any contacts listed under it)